layouts/_default/single.html

```
@@ -65,6 +124,176 @@ <h1 class="post-title">
+        <circle cx="18" cy="5" r="3"/><circle cx="6" cy="12" r="3"/><circle cx="18" cy="19" r="3"/>
+        <line x1="8.59" y1="13.51" x2="15.42" y2="17.49"/><line x1="15.41" y1="6.51" x2="8.59" y2="10.49"/>
+      </svg>
+      <span>{{ i18n "tools.share" | default "Share" | safeHTML }}</span>
+    </button>
+  </div>
+
+  <div class="abs-panels">
+
+    {{/* Panel 1: TOC */}}
+    <div class="abs-panel" id="abs-panel-toc" role="tabpanel" aria-labelledby="abs-tab-toc">
+      {{- $headers := findRE "<h[1-6].*?>(.|\n])+?</h[1-6]>" .Content -}}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)